certified-connectors/E-goi/apiProperties.json

```diff
@@ -0,0 +1,32 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```

Comment:
Unfortunately, pure white (#FFFFFF) isn't an allowed color for the icon. It can be off-white if you would like.